### ISSUE TITLE
migrate site to web1

### ISSUE THREAD

**Issue by edunham** [OSULUG/OSULUG-Website#39]
It's currently on twist.o.o, who's slated for decommission. Move the site, set it up in a virtualenv, and twist Jenkins's arm into updating on a github post-commit hook.

**Comment by edunham** [comment]
@ElijahCaine is this still a thing? can this turn into "host on gh pages" or something?

**Comment by ElijahCaine** [comment]
@edunham I'd be up for turning this into a GH pages site. @lucywyman thoughts?

**Comment by lucywyman** [comment]
Yeah, that seems easiest. I can get that done once I finish this pesky OS2 assignment!

**Comment by edunham** [comment]
@lucywyman ping, is this still a thing?

**Comment by lucywyman** [comment]
It could be, but is not something I've devoted thought to since November.  Should it be?

**Comment by edunham** [comment]
No idea; I've been gone from lug for over a year at this point. @ElijahCaine , pls use presidential decision powers here.

**Comment by ElijahCaine** [comment]
@edunham I'm putting 'Figure out where to host lug.oregonstate.edu' on the slate for this coming term. Thanks for the reminder.
